lib/workers/repository/extract/index.ts

```diff
@@ -47,6 +47,13 @@ export async function extractAllDependencies(
   const extractResults = await Promise.all(
     extractList.map(async (managerConfig) => {
       const packageFiles = await getManagerPackageFiles(managerConfig);
+      for (const p of packageFiles) {
+        p.deps?.forEach((dep) => {
+          if (!!config.updateInternalDeps && !dep.isInternal) {
```

Comment:
```suggestion
          if (!config.updateInternalDeps && dep.isInternal) {
```

Shouldn't it be like this? If updateInternalDeps=fasle and it's an internal dep then skip it?